style.css

```diff
@@ -0,0 +1,80 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@600;700&family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  margin: 0;
+  color: white;
+}
+
+.nav-bar {
+  background-color: #3e3c3c;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 0 10%;
+  height: 72px;
+}
+
+h2 {
+  margin-right: auto;
+  font-size: 18px;
+  font-family: 'Roboto', sans-serif;
+}
+
+#btn-menu {
+  font-size: 18px;
+  padding-left: 2%;
+}
+
+h1 {
+  font-size: 56px;
+  font-family: 'Crete Round', serif;
+  color: #ff6b00;
+}
+
+h3 {
+  font-size: 24px;
+  font-family: 'Roboto', sans-serif;
+}
+
+p {
+  font-size: 16px;
+}
+
+.container {
+  width: auto;
+}
+
+.hero {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  background-color: #1c1a19;
+  background-image: url("icons/images/illustration-header-mobile.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
```

Comment:
- [ ] Kindly you may consider moving the background image a little bit down as it is the case on the template.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212110140-29963f2a-61e3-4d17-8a71-a90342c12211.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212110304-c0bc23a9-ea73-4b3e-a62e-e60043c7d160.png)